warehouse/query-core/src/test/java/datawave/query/jexl/visitors/ValidComparisonVisitorTest.java

```diff
@@ -78,7 +78,7 @@ public void testInvalidGE() throws ParseException {
         ValidComparisonVisitor.check(script);
     }
     
-    @Test(expected = DatawaveFatalQueryException.class)
+    @Test(expected = ParseException.class)
```

Comment:
Do we not want a DatawaveFatalQueryException ?

Reply:
In the old code it failed because datawave didn't allow it.  In the new code it fails because JEXL doesn't allow it.